roles/ansible-runner/tasks/runner.yml

```diff
@@ -0,0 +1,49 @@
+- name: Download source
+  git:
+    repo: "{{ item.repo }}"
+    dest: "/opt/source/{{ item.name }}"
+    update: true
+
+- name: Install config file
+  template:
+    src: ansible-runner-config
+    dest: /etc/default/{{ item.name }}
+    mode: 0600
+    owner: "{{ item.user }}"
```

Comment:
This was copied/pasetd from main.yml, i suppose the ordering just happened?  Will update here, tho